src/Build/Logging/ConsoleLogger.cs

```diff
@@ -477,6 +475,30 @@ public void CustomEventHandler(object sender, CustomBuildEventArgs e)
             _consoleLogger.CustomEventHandler(sender, e);
         }
 
+        /// <summary>
+        /// Returns the minimum importance of messages logged by this logger.
+        /// </summary>
+        /// <returns>
+        /// The minimum message importance corresponding to this logger's verbosity or (MessageImportance.High - 1)
+        /// if this logger does not log messages of any importance.
+        /// </returns>
+        internal MessageImportance GetMinimumMessageImportance()
+        {
+            if (Verbosity >= BaseConsoleLogger.ImportanceToMinimumVerbosity(MessageImportance.Low, out _))
+            {
+                return MessageImportance.Low;
+            }
+            else if (Verbosity >= BaseConsoleLogger.ImportanceToMinimumVerbosity(MessageImportance.Normal, out _))
+            {
+                return MessageImportance.Normal;
+            }
+            else if (Verbosity >= BaseConsoleLogger.ImportanceToMinimumVerbosity(MessageImportance.High, out _))
+            {
+                return MessageImportance.High;
+            }
+            return MessageImportance.High - 1;
```

Comment:
Not sure I understand this; comment?

Reply:
It is explained in the method doc-comment, I've added an inline comment right here also.

I was debating adding an internal constant with this value but have a hard time coming up with a good name (and type where to declare it for that matter). It's basically a value representing the minimum message importance logged by quiet loggers, i.e. it's the non-existent "higher than high".

Reply:
Critical?

Reply:
That would be the name if it was a real MessageImportance value but it is just a marker saying "not interested in messages regardless of importance".